devtools/cli/src/main/java/io/quarkus/cli/common/DataOptions.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.cli.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import picocli.CommandLine;
+
+public class DataOptions {
+
+    public Map<String, String> data = new HashMap<>();
+
+    @CommandLine.Option(names = "--data", mapFallbackValue = "", description = "Additional data")
```

Comment:
What is the difference between --data and -D used in other commands to pass key/values in?

Reply:
never mind - just realizing this happens at create time where -D already are present.